polo-common/src/main/java/dev/dhdf/polo/webclient/WebClient.java

```diff
@@ -153,7 +313,7 @@ public JSONObject doRequest(String method, String endpoint, String body, Boolean
                     return null;
                 }
             } else {
-                logger.error("An invalid endpoint was called for.");
+                logger.error("An invalid endpoint " + endpoint + " was called for.");
```

Comment:
Should use message formatting here. Bukkit's logger doesn't have a message formatter, but our slf4j one does.

```suggestion
                logger.error("An invalid endpoint {} was called for.", endpoint);
```